node-notes-cli/app.js

```diff
@@ -0,0 +1,37 @@
+const dataModel = require('./data.json');
+const fs = require('fs');
+
+for (const key in dataModel.notes) {
+  if (process.argv[2] === 'read') {
+    console.log(`${key}: ${dataModel.notes[key]}`);
+  }
+}
+
+if (process.argv[2] === 'create') {
+  const key = dataModel.nextId;
+  dataModel.notes[key] = process.argv[3];
+  dataModel.nextId++;
+  const dataObject = JSON.stringify(dataModel, null, 2);
+  fs.writeFile('data.json', dataObject, 'utf8', err => {
+    if (err) throw err;
+  });
+}
+
+for (const key in dataModel.notes) {
+  if (process.argv[2] === 'delete' && key === process.argv[3]) {
+    delete dataModel.notes[key];
+    const dataObject = JSON.stringify(dataModel, null, 2);
+    fs.writeFile('data.json', dataObject, 'utf8', err => {
+      if (err) throw err;
+    });
+  }
+}
```

Comment:
You can access the note directly using bracket notation without the loop.

Reply:
![Screen Shot 2021-03-12 at 5 47 12 PM](https://user-images.githubusercontent.com/74885918/111014931-1acf7c00-835b-11eb-9bd0-b16b1b02b0d1.png)